### PR TITLE
Spelling: close → closed, store, upgrade, new version

### PR DESCRIPTION
**Pull request by comradekingu**
I think "appery" is a whole lot better than "app store" "appstore" or "store". In the same vein as "winery".

### CODE REVIEW

File: js/components/root.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
 		template: '' +
 		'<div class="section" id="caniupdate">' +
 		'	<h2 class="inlineblock">' +
-		'		'+ t('caniupdate', 'Can I update?') + ' {{ canUpdate }}' +
+		'		'+ t('caniupdate', 'Can I upgrade?') + ' {{ canUpdate }}' +
```
Comment (nickvergessen) [inline js/components/root.js:18]: In Nextcloud we call this update

File: js/components/root.js [hunk 2]
```diff
@@ -25,13 +25,13 @@
 		'	<template v-if="missing.length">' +
 		'		<h3>' + t('caniupdate', 'Missing updates') + '</h3>' +
 		'		<ul>' +
-		'			<li v-for="app in missing">{{app.appName}} <a :href="\'https://apps.nextcloud.com/apps/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
+		'			<li v-for="app in missing">{{app.appName}} <a :href="\'https://apps.nextcloud.com/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
```
Comment (nickvergessen) [inline js/components/root.js:28]: you are breaking the URL

File: js/components/root.js [hunk 3]
```diff
@@ -25,13 +25,13 @@
...
 		'		</ul>'+
 		'	</template>' +
 		'	<template v-if="available.length">' +
-		'		<h3>' + t('caniupdate', 'Available updates') + '</h3>' +
+		'		<h3>' + t('caniupdate', 'Available new versions') + '</h3>' +
 		'		<ul>' +
-		'			<li v-for="app in available">{{app.appName}} <a :href="\'https://apps.nextcloud.com/apps/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
+		'			<li v-for="app is available">{{app.appName}} <a :href="\'https://apps.nextcloud.com/\' + app.appId">' + t('caniupdate', 'View in store') + ' ↗</a></li>' +
```
Comment (nickvergessen) [inline js/components/root.js:34]: the `is` is JS magic syntax hence you break the listing of the updates....

Comment (nickvergessen) [reply js/components/root.js:34]: you are breaking the URL by removing the `apps/` part

File: js/components/root.js [hunk 4]
```diff
@@ -95,15 +95,15 @@
 				}
 
 				if (this.alreadyOnLatest) {
-					return t('caniupdate', 'This app will show results again, once Nextcloud 14 is close.');
+					return t('caniupdate', 'This app will show results again, once Nextcloud 14 is closed.');
```
Comment (nickvergessen) [inline js/components/root.js:98]: `Close` as in `close to be done`.
Not as a word like `finished`

File: js/components/root.js [hunk 5]
```diff
@@ -95,15 +95,15 @@
...
 				}
 
 				if (this.appstoreDisabled) {
 					return t('caniupdate', 'Please make sure your config.php does not set <samp>appstoreenabled</samp> to false.');
 				}
 
 				if (this.appstoreFailed) {
-					return t('caniupdate', 'Could not connect to the appstore or the appstore returned no updates at all. Search manually for updates or make sure your server has access to the internet and can connect to the appstore.');
+					return t('caniupdate', 'Could not connect to the store or it returned no upgrades at all. Search manually for updates or make sure your server has access to the Internet and can connect to the store.');
```
Comment (nickvergessen) [inline js/components/root.js:106]: Wrong words used again

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Comment by nickvergessen** [comment]
Since the only remaining point it internet with a capital I, which I'm also not sure about, I'm closing this.

File: js/components/root.js (hunk 4 above)
Comment (comradekingu) [reply js/components/root.js:98]: I am thinking others, like me, will not catch that, in turn making for erroneous translations.
Would you accept "nearing release"?

File: js/components/root.js (hunk 2 above)
Comment (comradekingu) [reply js/components/root.js:28]: That URL doesn't work for me. Logged in or not.

File: js/components/root.js (hunk 3 above)
Comment (comradekingu) [reply js/components/root.js:34]: Could you explain "app in available" or what happens to it?

File: js/components/root.js (hunk 5 above)
Comment (comradekingu) [reply js/components/root.js:106]: In the very least it should be "app store" like elsewhere. I thought "appery" to be a good word. "app store" has a fairly ugly connotation in that it likens it to the very worst of software distribution.

Edit: "Internet" is in fairly consistent use throughout Nextcloud. Trying to go for the version that produces the fewest errors in translation.
https://en.wikipedia.org/wiki/Capitalization_of_%22Internet%22

There are only a few uses of what i would at-least call non-historic uses of "update" left. Wanting to distinguish between a program being in the process of updating and upgrading i would argue is a meaningful use of language for what amounts to current differences. Chosing there to err on the side of functionality, while "upgrade" in that respect is the least problematic of the two, I've found using "new version" avoids any and all difficulty.

You also need a distinction between "refresh" and "update" to use the latter for software upgrades, which is not present in some languages.

File: js/components/root.js (hunk 3 above)
Comment (nickvergessen) [reply js/components/root.js:34]: https://vuejs.org/v2/guide/list.html#v-for-with-an-Object

File: js/components/root.js (hunk 2 above)
Comment (nickvergessen) [reply js/components/root.js:28]: All links on the appstore contain `apps/`:
https://apps.nextcloud.com/apps/contacts

File: js/components/root.js (hunk 1 above)
Comment (comradekingu) [reply js/components/root.js:18]: 14 "upgrade"s, comparable number "update"-upgrades.

**Pull request closed without merging**